src/minter-suite-mapping.ts

```diff
@@ -153,6 +155,38 @@ export function handlePurchaseToDisabledUpdated(
   project.save();
 }
 
+// IFilteredMinterV2 events
+export function handleProjectMaxInvocationsLimitUpdated(
+  event: ProjectMaxInvocationsLimitUpdated
+): void {
+  let minterProjectAndConfig = loadMinterProjectAndConfig(
+    event.address,
+    event.params._projectId,
+    event.block.timestamp
+  );
+
+  if (!minterProjectAndConfig) {
+    return;
+  }
+
+  let project = minterProjectAndConfig.project;
+  let projectMinterConfig = minterProjectAndConfig.projectMinterConfiguration;
+
+  if (event.params._maxInvocations > project.maxInvocations) {
+    log.warning(
```

Comment:
optional - might be worth an in-line comment that says we don't expect minter contracts to allow this to happen

Reply:
this was addressed in Ben Y's review, so resolving

Reply:
based on the conversation above with @yoshiwarab , I actually did replace this entire conditional with a @dev comment